src/test/java/selenium/tutorial/studies/DoubleClickHandlingTest.java

```diff
@@ -0,0 +1,81 @@
+package selenium.tutorial.studies;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.*;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.interactions.Actions;
+import org.testng.Assert;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+
+import com.gargoylesoftware.css.dom.Property;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+
+public class DoubleClickHandlingTest {
+	
+	WebDriver driver;
+	
+	
+	
+	@BeforeTest
+	public void setup() throws IOException {
+		InputStream input= DoubleClickHandlingTest.class.getClassLoader().getResourceAsStream("TestData.properties");
+		Properties property = new Properties();
+		property.load(input);
+		
+		if ("chrome".equalsIgnoreCase(property.getProperty("activeBrowser"))) {
+			
+			WebDriverManager.chromedriver().version(property.getProperty("chromeDriverVersion")).setup();
+			driver = new ChromeDriver();
+			driver.manage().window().maximize();	
```

Comment:
driver.manage().window().maximize() is common in both(if and else) blocks . In stead of writing in two blocks ,  keep it outside of each blocks so that both blocks can execute same statement. No of lines in the code will be reduced

Reply:
can we add deletAllCookies() as well. As it will be same across all browser drivers, keep it in a common place rathan adding in each different blocks